### PR TITLE
Working Gem

### PR DESCRIPTION
**Pull request by Triodan**
The gem is now functional on a basic level, allowing for create, read, update and delete operations on a simple SQLite database model.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# CrudWrap
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/crud_wrap`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+TODO: Delete this and the text above, and describe your gem
```
Comment (miduddin) [inline README.md]: ini dilengkapin dulu ya dokumentasinya

File: spec/crud_wrap_spec.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    expect(File).to exist(TEST_DATABASE_FILENAME)
+  end
+
+  after :all do
+    cleanup
+  end
+
+end
+
+RSpec.describe Database do
```
Comment (miduddin) [inline spec/crud_wrap_spec.rb]: kalo pake rspec biasanya nama file specnya nyesuaiin dengan file yg di-test, misal `lib/crud_wrap/database.rb -> spec/crud_wrap/database_spec.rb`

Comment (miduddin) [reply spec/crud_wrap_spec.rb]: path file barunya salah, harusnya kayak yg saya kasih tau di comment sebelumnya. Terus semua file source `*.rb` harusnya ada spec-nya masing2, ga cuma `database.rb`.

File: spec/database_spec.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  it 'stores records into the database' do
+    l = Lapak.new('Toko Andi', 'Andi', 199)
+    expect(@db.create(l)).to eq []
+
+    m = Lapak.new('Toko Budi', 'Budi', 83)
+    expect(@db.create(m)).to eq []
+  end
+
+  it 'fetches all records from the database' do
+    expect(@db.read_all.length).to eq 2
```
Comment (miduddin) [inline spec/database_spec.rb]: Tiap example case (`it` block) sebaiknya bisa di-run secara individual (i.e. independent terhadap example2 lain), biar kalo mau nge-run 1 example case doang bisa cepet (meskipun kalo di sini cepet2 aja sih).

Di contoh ini, kalo saya nge-run `bundle exec rspec spec/database_spec.rb:22` bakal error.

File: spec/database_spec.rb [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
```
Comment (miduddin) [inline spec/database_spec.rb]: https://github.com/Triodan/crud_wrap/pull/1#discussion_r209534115

Comment (Triodan) [reply spec/database_spec.rb]: fixed

**Pull request merged**